test/cpp/end2end/shutdown_test.cc

```diff
@@ -83,7 +84,8 @@ class ShutdownTest : public ::testing::Test {
     grpc::string server_address = "localhost:" + to_string(port);
 
     ServerBuilder builder;
-    builder.AddListeningPort(server_address, InsecureServerCredentials());
+    auto server_creds = GetServerCredentials(GetParam());
```

Comment:
Yes, my local repo needs sync. Added GetCredentialsProvider() to all test_credentials_provider api. PTAL!